[2022-08-29 15:46:55,970] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: h1b_lca_data_v3.entry_task scheduled__2021-01-10T06:00:00+00:00 [queued]>
[2022-08-29 15:46:55,986] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: h1b_lca_data_v3.entry_task scheduled__2021-01-10T06:00:00+00:00 [queued]>
[2022-08-29 15:46:55,988] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-08-29 15:46:55,989] {taskinstance.py:1239} INFO - Starting attempt 26 of 26
[2022-08-29 15:46:55,990] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-08-29 15:46:56,006] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): entry_task> on 2021-01-10 06:00:00+00:00
[2022-08-29 15:46:56,014] {standard_task_runner.py:52} INFO - Started process 12051 to run task
[2022-08-29 15:46:56,019] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'h1b_lca_data_v3', 'entry_task', 'scheduled__2021-01-10T06:00:00+00:00', '--job-id', '1534', '--raw', '--subdir', 'DAGS_FOLDER/upload_h1b_data_to_gcp.py', '--cfg-path', '/tmp/tmplvpa__ye', '--error-file', '/tmp/tmpuz4q5tg2']
[2022-08-29 15:46:56,022] {standard_task_runner.py:77} INFO - Job 1534: Subtask entry_task
[2022-08-29 15:46:56,105] {logging_mixin.py:109} INFO - Running <TaskInstance: h1b_lca_data_v3.entry_task scheduled__2021-01-10T06:00:00+00:00 [running]> on host efb24f1b01c6
[2022-08-29 15:46:56,188] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=h1b_lca_data_v3
AIRFLOW_CTX_TASK_ID=entry_task
AIRFLOW_CTX_EXECUTION_DATE=2021-01-10T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-01-10T06:00:00+00:00
[2022-08-29 15:46:56,192] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-29 15:46:56,193] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo 0']
[2022-08-29 15:46:56,213] {subprocess.py:85} INFO - Output:
[2022-08-29 15:46:56,217] {subprocess.py:89} INFO - 0
[2022-08-29 15:46:56,218] {subprocess.py:93} INFO - Command exited with return code 0
[2022-08-29 15:46:56,252] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=h1b_lca_data_v3, task_id=entry_task, execution_date=20210110T060000, start_date=20220829T154655, end_date=20220829T154656
[2022-08-29 15:46:56,315] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-08-29 15:46:56,359] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
